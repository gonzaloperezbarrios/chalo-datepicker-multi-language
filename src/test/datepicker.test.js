import React from "react";
import renderer from "react-test-renderer";
import InputDatePicker from "../utils/main.js";

test("InputDatePicker basic snapshot", () => {
  const component = renderer.create(
    <InputDatePicker
      language="en"
      dateDefault={new Date(new Date("09-02-2019"))}
      handleChange={console.log}
    />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
