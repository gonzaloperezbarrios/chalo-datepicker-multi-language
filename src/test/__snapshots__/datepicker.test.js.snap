// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputDatePicker basic snapshot 1`] = `
<div
  className="DayPickerInput"
>
  <input
    onBlur={[Function]}
    onChange={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    placeholder="MM/DD/YYYY"
    value="09/02/2019"
  />
</div>
`;
